docs/changelogs/v0.21.md

```diff
@@ -6,13 +6,24 @@
 
 - [Overview](#overview)
 - [🔦 Highlights](#-highlights)
+  - [Accelerated DHT Client is no longer experimental](#--empty-repo-is-now-the-default)
 - [📝 Changelog](#-changelog)
 - [👨‍👩‍👧‍👦 Contributors](#-contributors)
 
 ### Overview
 
 ### 🔦 Highlights
 
+#### Accelerated DHT Client is no longer experimental
+
+The accelerated DHT client is now the main recomended solution for peoples who
+are hosting lots of data.
```

Comment:
```suggestion
are hosting lots of data.  By trading some upfront DHT caching and increased memory usage, one gets provider latency improvements of Xx.  See [the docs](docs/config.md#routingaccelerateddhtclient) for more info.
```

Reply:
I'll remove
> one gets provider latency improvements of Xx

it's not the reason we are telling people to use it, it's mainly throughput